### PR TITLE
Some minor upgrade on gems and examples

### PR DESCRIPTION
**Pull request by rafaelfranca**

### CODE REVIEW

File: script/bootstrap [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+bundle install --binstubs --path vendor --without production
```
Comment (joelmoss) [inline script/bootstrap]: Just bundle install will do here. Using binstubs and setting the path are pretty specific settings, neither of which are common. I also tend to prefer a `rake setup` task, but this bootstrap script is fine. thx

Comment (rafaelfranca) [inline script/bootstrap]: Ok then. We always use binstubs and path install in ours projects. But I'll change it.

Should I remove the `--without production` too?

Comment (joelmoss) [inline script/bootstrap]: Yeah, just the standard `bundle install` will do. thx.

**Comment by rafaelfranca** [comment]
Done.

**Comment by joelmoss** [comment]
Cheers!

**Pull request merged**